### DISCUSSION TITLE
Run samples?

### DISCUSSION THREAD

**Discussion by Chicagoan2016** [MarimerLLC/csla#3784] [Questions]
Folks,  Whenever I download (any of the) Samples. None of the nuget packages work. I have to manually remove and re-add them or sometimes just create a new project and add files to it. That's after trying 'Package restore' settings under Tools in Visual studio. Is it just me? : )
Regards

**Comment by rockfordlhotka** [comment]
That is odd. I haven't updated most of them to CSLA 8 yet, so they should still be targeting 7, or maybe a prerelease of 8.

**Reply by Chicagoan2016** [reply]
The Winsortfilter sample I downloaded targetted prerelease of 8.
